### PR TITLE
fix(plasma-web-docs): Remove button blur prop from component & docs

### PR DESCRIPTION
**Pull request by TitanKuzmich**
## What/why Changed

Blur свойство теперь deprecated для компонента Button. Работает только на кнопках с прозрачностью

**Пример:**
image
image

**Было в доке**

image

**Стало**

image

-------

<details>
  <summary>:baby_chick: Download canary assets:</summary>
  <blockquote>
    <details>
      <summary>borderRadius</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_default_react-native--canary.785.6381248900.ts'>borderRadius_default_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_b2c_react-native--canary.785.6381248900.ts'>borderRadius_plasma_b2c_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_web_react-native--canary.785.6381248900.ts'>borderRadius_plasma_web_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberHealth_react-native--canary.785.6381248900.ts'>borderRadius_sberHealth_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_business_react-native--canary.785.6381248900.ts'>borderRadius_sbermarket_business_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_metro_react-native--canary.785.6381248900.ts'>borderRadius_sbermarket_metro_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_react-native--canary.785.6381248900.ts'>borderRadius_sbermarket_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_selgros_react-native--canary.785.6381248900.ts'>borderRadius_sbermarket_selgros_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_wlbusiness_react-native--canary.785.6381248900.ts'>borderRadius_sbermarket_wlbusiness_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberonline_react-native--canary.785.6381248900.ts'>borderRadius_sberonline_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberprime_react-native--canary.785.6381248900.ts'>borderRadius_sberprime_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_stylesSalute_react-native--canary.785.6381248900.ts'>borderRadius_stylesSalute_react-native--canary.785.6381248900.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>color</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_ios-swift--canary.785.6381248900.swift'>color_default_ios-swift--canary.785.6381248900.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_kotlin--canary.785.6381248900.kt'>color_default_kotlin--canary.785.6381248900.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_react-native--canary.785.6381248900.ts'>color_default_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_xml--canary.785.6381248900.xml'>color_default_xml--canary.785.6381248900.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_ios-swift--canary.785.6381248900.swift'>color_plasma_b2c_ios-swift--canary.785.6381248900.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_kotlin--canary.785.6381248900.kt'>color_plasma_b2c_kotlin--canary.785.6381248900.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_react-native--canary.785.6381248900.ts'>color_plasma_b2c_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_xml--canary.785.6381248900.xml'>color_plasma_b2c_xml--canary.785.6381248900.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_ios-swift--canary.785.6381248900.swift'>color_plasma_web_ios-swift--canary.785.6381248900.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_kotlin--canary.785.6381248900.kt'>color_plasma_web_kotlin--canary.785.6381248900.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_react-native--canary.785.6381248900.ts'>color_plasma_web_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_xml--canary.785.6381248900.xml'>color_plasma_web_xml--canary.785.6381248900.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.785.6381248900.swift'>color_sberHealth_ios-swift--canary.785.6381248900.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.785.6381248900.kt'>color_sberHealth_kotlin--canary.785.6381248900.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.785.6381248900.ts'>color_sberHealth_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_xml--canary.785.6381248900.xml'>color_sberHealth_xml--canary.785.6381248900.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.785.6381248900.swift'>color_sbermarket_business_ios-swift--canary.785.6381248900.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.785.6381248900.kt'>color_sbermarket_business_kotlin--canary.785.6381248900.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.785.6381248900.ts'>color_sbermarket_business_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_xml--canary.785.6381248900.xml'>color_sbermarket_business_xml--canary.785.6381248900.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.785.6381248900.swift'>color_sbermarket_ios-swift--canary.785.6381248900.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.785.6381248900.kt'>color_sbermarket_kotlin--canary.785.6381248900.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.785.6381248900.swift'>color_sbermarket_metro_ios-swift--canary.785.6381248900.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.785.6381248900.kt'>color_sbermarket_metro_kotlin--canary.785.6381248900.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.785.6381248900.ts'>color_sbermarket_metro_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_xml--canary.785.6381248900.xml'>color_sbermarket_metro_xml--canary.785.6381248900.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.785.6381248900.ts'>color_sbermarket_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.785.6381248900.swift'>color_sbermarket_selgros_ios-swift--canary.785.6381248900.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.785.6381248900.kt'>color_sbermarket_selgros_kotlin--canary.785.6381248900.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.785.6381248900.ts'>color_sbermarket_selgros_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_xml--canary.785.6381248900.xml'>color_sbermarket_selgros_xml--canary.785.6381248900.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.785.6381248900.swift'>color_sbermarket_wlbusiness_ios-swift--canary.785.6381248900.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.785.6381248900.kt'>color_sbermarket_wlbusiness_kotlin--canary.785.6381248900.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.785.6381248900.ts'>color_sbermarket_wlbusiness_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_xml--canary.785.6381248900.xml'>color_sbermarket_wlbusiness_xml--canary.785.6381248900.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_xml--canary.785.6381248900.xml'>color_sbermarket_xml--canary.785.6381248900.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.785.6381248900.swift'>color_sberonline_ios-swift--canary.785.6381248900.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.785.6381248900.kt'>color_sberonline_kotlin--canary.785.6381248900.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.785.6381248900.ts'>color_sberonline_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_xml--canary.785.6381248900.xml'>color_sberonline_xml--canary.785.6381248900.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.785.6381248900.swift'>color_sberprime_ios-swift--canary.785.6381248900.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.785.6381248900.kt'>color_sberprime_kotlin--canary.785.6381248900.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.785.6381248900.ts'>color_sberprime_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_xml--canary.785.6381248900.xml'>color_sberprime_xml--canary.785.6381248900.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_ios-swift--canary.785.6381248900.swift'>color_stylesSalute_ios-swift--canary.785.6381248900.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_kotlin--canary.785.6381248900.kt'>color_stylesSalute_kotlin--canary.785.6381248900.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_react-native--canary.785.6381248900.ts'>color_stylesSalute_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_xml--canary.785.6381248900.xml'>color_stylesSalute_xml--canary.785.6381248900.xml</a><br>
      </blockquote>
    </details>
    <details>
      <summary>shadow</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_default_react-native--canary.785.6381248900.ts'>shadow_default_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_b2c_react-native--canary.785.6381248900.ts'>shadow_plasma_b2c_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_web_react-native--canary.785.6381248900.ts'>shadow_plasma_web_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberHealth_react-native--canary.785.6381248900.ts'>shadow_sberHealth_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_business_react-native--canary.785.6381248900.ts'>shadow_sbermarket_business_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_metro_react-native--canary.785.6381248900.ts'>shadow_sbermarket_metro_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.785.6381248900.ts'>shadow_sbermarket_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_selgros_react-native--canary.785.6381248900.ts'>shadow_sbermarket_selgros_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_wlbusiness_react-native--canary.785.6381248900.ts'>shadow_sbermarket_wlbusiness_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberonline_react-native--canary.785.6381248900.ts'>shadow_sberonline_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberprime_react-native--canary.785.6381248900.ts'>shadow_sberprime_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_stylesSalute_react-native--canary.785.6381248900.ts'>shadow_stylesSalute_react-native--canary.785.6381248900.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>spacing</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_default_react-native--canary.785.6381248900.ts'>spacing_default_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_b2c_react-native--canary.785.6381248900.ts'>spacing_plasma_b2c_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_web_react-native--canary.785.6381248900.ts'>spacing_plasma_web_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberHealth_react-native--canary.785.6381248900.ts'>spacing_sberHealth_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_business_react-native--canary.785.6381248900.ts'>spacing_sbermarket_business_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_metro_react-native--canary.785.6381248900.ts'>spacing_sbermarket_metro_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_react-native--canary.785.6381248900.ts'>spacing_sbermarket_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_selgros_react-native--canary.785.6381248900.ts'>spacing_sbermarket_selgros_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_wlbusiness_react-native--canary.785.6381248900.ts'>spacing_sbermarket_wlbusiness_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberonline_react-native--canary.785.6381248900.ts'>spacing_sberonline_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberprime_react-native--canary.785.6381248900.ts'>spacing_sberprime_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_stylesSalute_react-native--canary.785.6381248900.ts'>spacing_stylesSalute_react-native--canary.785.6381248900.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>typo</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.785.6381248900.swift'>typo_mage_ios-swift--canary.785.6381248900.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.785.6381248900.kt'>typo_mage_kotlin--canary.785.6381248900.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.785.6381248900.ts'>typo_mage_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.785.6381248900.swift'>typo_plasma_ios-swift--canary.785.6381248900.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.785.6381248900.kt'>typo_plasma_kotlin--canary.785.6381248900.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.785.6381248900.ts'>typo_plasma_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.785.6381248900.swift'>typo_ruler_ios-swift--canary.785.6381248900.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.785.6381248900.kt'>typo_ruler_kotlin--canary.785.6381248900.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.785.6381248900.ts'>typo_ruler_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.785.6381248900.swift'>typo_sage_ios-swift--canary.785.6381248900.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.785.6381248900.kt'>typo_sage_kotlin--canary.785.6381248900.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.785.6381248900.ts'>typo_sage_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.785.6381248900.swift'>typo_sbermarket_ios-swift--canary.785.6381248900.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.785.6381248900.kt'>typo_sbermarket_kotlin--canary.785.6381248900.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.785.6381248900.ts'>typo_sbermarket_react-native--canary.785.6381248900.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.785.6381248900.swift'>typo_soulmate_ios-swift--canary.785.6381248900.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.785.6381248900.kt'>typo_soulmate_kotlin--canary.785.6381248900.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.785.6381248900.ts'>typo_soulmate_react-native--canary.785.6381248900.ts</a><br>
      </blockquote>
    </details>
  </blockquote>
</details>

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.236.2-canary.785.6381248900.0
  npm install @salutejs/plasma-core@1.134.2-canary.785.6381248900.0
  npm install @salutejs/plasma-hope@1.235.2-canary.785.6381248900.0
  npm install @salutejs/plasma-icons@1.165.2-canary.785.6381248900.0
  npm install @salutejs/plasma-new-hope@0.5.2-canary.785.6381248900.0
  npm install @salutejs/plasma-temple@1.183.2-canary.785.6381248900.0
  npm install @salutejs/plasma-ui@1.215.2-canary.785.6381248900.0
  npm install @salutejs/plasma-web@1.236.2-canary.785.6381248900.0
  npm install @salutejs/plasma-sb-utils@0.132.2-canary.785.6381248900.0
  # or 
  yarn add @salutejs/plasma-b2c@1.236.2-canary.785.6381248900.0
  yarn add @salutejs/plasma-core@1.134.2-canary.785.6381248900.0
  yarn add @salutejs/plasma-hope@1.235.2-canary.785.6381248900.0
  yarn add @salutejs/plasma-icons@1.165.2-canary.785.6381248900.0
  yarn add @salutejs/plasma-new-hope@0.5.2-canary.785.6381248900.0
  yarn add @salutejs/plasma-temple@1.183.2-canary.785.6381248900.0
  yarn add @salutejs/plasma-ui@1.215.2-canary.785.6381248900.0
  yarn add @salutejs/plasma-web@1.236.2-canary.785.6381248900.0
  yarn add @salutejs/plasma-sb-utils@0.132.2-canary.785.6381248900.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-785/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-785/
b2c storybook: http://plasma.sberdevices.ru/pr/pr-785/b2c-storybook/
web storybook: http://plasma.sberdevices.ru/pr/pr-785/web-storybook/
ui storybook: http://plasma.sberdevices.ru/pr/pr-785/ui-storybook/
temple storybook: http://plasma.sberdevices.ru/pr/pr-785/temple-storybook/

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ npx lerna bootstrap --scope [имя пакета 1] --scope [имя пакета
 Если возникла какая-либо проблема со сборкой, можно попробовать полностью удалить все зависимости и установить их заново:
 
 ```sh
-rm -rm ./node_modules/
+rm -rf ./node_modules/
```
Comment (Yakutoc) [inline CONTRIBUTING.md:43]: 👍

File: packages/plasma-core/api/plasma-core.api.md [hunk 2]
```diff
@@ -289,7 +289,7 @@ export const buttonFocused = "var(--plasma-colors-button-focused)";
 export const buttonPrimary = "var(--plasma-colors-button-primary)";
 
 // @public
-export interface ButtonProps<T = HTMLElement> extends Partial<PinProps>, FocusProps, OutlinedProps, DisabledProps, ShiftProps, BlurProps, AsProps, Omit<React.AnchorHTMLAttributes<T>, 'type'>, React.ButtonHTMLAttributes<T> {
+export interface ButtonProps<T = HTMLElement> extends Partial<PinProps>, FocusProps, OutlinedProps, DisabledProps, ShiftProps, AsProps, Omit<React.AnchorHTMLAttributes<T>, 'type'>, React.ButtonHTMLAttributes<T> {
```
Comment (Yakutoc) [inline packages/plasma-core/api/plasma-core.api.md:292]: @TitanKuzmich 

Подскажи пожалуйста, а ты же в корне проекта запустил команду - `npm run api:report` ?

Comment (TitanKuzmich) [reply packages/plasma-core/api/plasma-core.api.md:292]: Да, запускал
Это сгенеренный файл

**Review by Yakutoc — APPROVED** [review]

**Review by kayman233 — APPROVED** [review]

**Review by neretin-trike — APPROVED** [review]

File: packages/plasma-core/src/components/Button/Button.types.ts [hunk 3]
```diff
@@ -28,6 +27,12 @@ export interface ButtonProps<T = HTMLElement>
      * Растягиваемость кнопки (кнопка занимает ширину контейнера)
      */
     stretch?: boolean;
+
+    /**
+     * Степень размытия фона
+     * @deprecated для кнопок без прозрачности не работает
+     */
+    blur?: keyof typeof blurs;
```
Comment (neretin-trike) [inline packages/plasma-core/src/components/Button/Button.types.ts:35]: кажется, что можно было deprecated комментарий оставить внутри `BlurProps`?

File: packages/plasma-core/src/components/Button/Button.types.ts [hunk 4]
```diff
@@ -141,9 +146,8 @@ export interface StyledButtonProps
         OutlinedProps,
         DisabledProps,
         ShiftProps,
-        BlurProps,
         ButtonIsContentProps,
-        Pick<ButtonProps, 'square' | 'stretch'> {}
+        Pick<ButtonProps, 'square' | 'stretch' | 'blur'> {}
```
Comment (neretin-trike) [inline packages/plasma-core/src/components/Button/Button.types.ts:150]: и тогда тут не придётся blur добавлять

File: packages/plasma-core/src/components/Button/Button.types.ts (hunk 3 above)
Comment (TitanKuzmich) [reply packages/plasma-core/src/components/Button/Button.types.ts:35]: BlurProps могут понадобиться в других компонентах и тогда в доке везде будет deprecated

Comment (neretin-trike) [reply packages/plasma-core/src/components/Button/Button.types.ts:35]: хм, звучит резонно, но на текущий момент я больше нигде не нашёл использования этого миксина 🤔 

image

оке, давай пока на это уровне оставим

Comment (neretin-trike) [reply packages/plasma-core/src/components/Button/Button.types.ts:35]: оке, давай пока на это уровне оставим

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-b2c@1.236.2-dev.0, @salutejs/plasma-core@1.134.2-dev.0, @salutejs/plasma-hope@1.235.2-dev.0, @salutejs/plasma-icons@1.165.2-dev.0, @salutejs/plasma-new-hope@0.5.2-dev.0, @salutejs/plasma-sb-utils@0.132.2-dev.0, @salutejs/plasma-temple-docs@0.173.2-dev.0, @salutejs/plasma-temple@1.183.2-dev.0, @salutejs/plasma-ui-docs@0.222.2-dev.0, @salutejs/plasma-ui@1.215.2-dev.0, @salutejs/plasma-web-docs@0.190.2-dev.0, @salutejs/plasma-web@1.236.2-dev.0, @salutejs/plasma-website@0.198.2-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-b2c@1.236.2-dev.0, @salutejs/plasma-core@1.134.2-dev.0, @salutejs/plasma-hope@1.235.2-dev.0, @salutejs/plasma-icons@1.165.2-dev.0, @salutejs/plasma-new-hope@0.5.2-dev.0, @salutejs/plasma-sb-utils@0.132.2-dev.0, @salutejs/plasma-temple-docs@0.173.2-dev.0, @salutejs/plasma-temple@1.183.2-dev.0, @salutejs/plasma-ui-docs@0.222.2-dev.0, @salutejs/plasma-ui@1.215.2-dev.0, @salutejs/plasma-web-docs@0.190.2-dev.0, @salutejs/plasma-web@1.236.2-dev.0, @salutejs/plasma-website@0.198.2-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-b2c@1.236.2-dev.0, @salutejs/plasma-core@1.134.2-dev.0, @salutejs/plasma-hope@1.235.2-dev.0, @salutejs/plasma-icons@1.165.2-dev.0, @salutejs/plasma-new-hope@0.5.2-dev.0, @salutejs/plasma-sb-utils@0.132.2-dev.0, @salutejs/plasma-temple-docs@0.173.2-dev.0, @salutejs/plasma-temple@1.183.2-dev.0, @salutejs/plasma-ui-docs@0.222.2-dev.0, @salutejs/plasma-ui@1.215.2-dev.0, @salutejs/plasma-web-docs@0.190.2-dev.0, @salutejs/plasma-web@1.236.2-dev.0, @salutejs/plasma-website@0.198.2-dev.0 🚀

**Pull request merged**